### ISSUE TITLE
How can I train for 2 classes?

### ISSUE THREAD

**Issue by isadoramsouza** [fabianbormann/Tensorflow_Segmentation#25]
I want to use the same dataset (pascal voc) for training, but change the number of classes. 
I only want 2 classes, background (black) and my object (white)...it's a binary segmentation.

thanks

**Comment by fabianbormann** [comment]
edit the images and reduce the classes (filters) from 21 to 1. Use sigmoid instead of softmax and binary_crossentropy in place of categorical
